lang/en/theme_boost_union.php

```diff
@@ -862,6 +862,74 @@
 $string['page3linkpositionboth'] = 'Add a link to the generic page 3 to the footnote and to the footer (questionmark) icon';
 $string['page3linkpositionsetting'] = 'Generic page 3 link position';
 $string['page3linkpositionsetting_desc'] = 'In this setting, you can configure if a link to the generic page 3 should be added automatically to the Moodle page. If you do not want to show a link automatically, you can add a link to {$a->url} from anywhere in Moodle manually.';
+// Settings: Accessibility tab.
+$string['accessibilitytab'] = 'Accessibility';
+// ... Section: Declaration of accessibility page.
+$string['accessibilityheading'] = 'Declaration of accessibility';
+// ... ... Setting: Enable accessibility page.
+$string['enableaccessibilitysetting'] = 'Enable Declaration of accessibility information page';
+$string['accessibilitydisabled'] = 'The declaration of accessibility information page is disabled for this site. There is nothing to see here.';
```

Comment:
Two different spellings - using "D" and "d" - are used on the settings page:
* here "The declaration of accessibility..."
* and later (e.g. L 874) "... shown on a Declaration of ..."

We should opt for one consistent stylization.